### ISSUE TITLE
[FEAT] Add possibility to add scopes and client-id, or any other options

### ISSUE THREAD

**Issue by RaphOb** [TikouWeb/gcp-switch-configuration#6]
Here an example : gcloud auth application-default login --scopes="https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/admin.directory.domain.readonly" --client-id-file=t.json

**Comment by bensolano** [comment]
yes and service account impersonnation as well please : gcloud auth application-default login --impersonate-service-account mysuperproject@itisamazing.iam.gserviceaccount.com
